test/libsolidity/syntaxTests/types/mapping/library_storage_parameter_with_mapping_1.sol

```diff
@@ -0,0 +1,6 @@
+library Test {
+    struct Nested { mapping(uint => uint)[2][] a; }
+    struct X { Nested n; }
+    function f(X storage x) public {}
+}
+// ----
```

Comment:
> `library_storage_parameter_with_mapping_1.sol`

How about `library_storage_parameter_with_nested_mapping.sol`?

Reply:
Yes, I like the idea. (I think now it is fine to keep all test together here, without moving anything to `iceRegressionTests/`.)